ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+
+- name: install Rust deps
+  apt:
+    name:
+      - pkg-config
+      - build-essential
+      - libssl-dev
+    state: present
+
+- name: install rustup in userspace for root
+  shell: |
+    set -eu
+    RUSTUP_VERSION=1.24.3
+    RUSTUP_SHA="a3cb081f88a6789d104518b30d4aa410009cd08c3822a1226991d6cf0442a0f8"
+    curl --proto '=https' --tlsv1.2 -sSf -O \
+      https://raw.githubusercontent.com/rust-lang/rustup/${RUSTUP_VERSION}/rustup-init.sh
+    echo "${RUSTUP_SHA}  rustup-init.sh" | sha256sum --check --
+    sh rustup-init.sh --default-toolchain nightly -y --component rust-src
+
+- name: Configure update script service
+  copy:
+    src: team_login/
+    dest: "{{ vars_team_login_path }}"
+
+- name: Build team login cron job
+  shell: "cd {{ vars_team_login_path }} && PATH=$PATH:$HOME/.cargo/bin cargo build"
+
+- name: Enable message of the day
+  template:
+    src: motd_sshd_config
+    dest: /etc/ssh/sshd_config.d/motd
+
+- name: Set up message of the day
+  template:
+    src: motd_rules
+    dest: /etc/motd
+
+- name: Install the team login binary
+  shell: "cp {{ vars_team_login_path }}/target/debug/team_login /etc/cron.team_login"
+
+- name: Set up the team login cron job
+  template:
+    src: crontab_append
+    dest: /etc/cron.d/team_login
+  # if the cron job is running right now, keep retrying until it finishes
+  register: task_result
+  until: task_result is not failed
+  retries: 10
+  delay: 5
+
+- name: Set up the files that initially appear in a user's home dir
+  copy:
+    src: skel/
+    dest: /etc/skel/
```

Comment:
I'm not too keen on the `skel` approach for providing pre-made scripts, as making any update/fix to them is going to be a pain for existing users on a machine. I'd directly add them to `/usr/local/bin`, that way they're centrally managed and we can add/update them without any issue.

Reply:
Note that you also moved the `config.toml` to `/usr/local/bin`, even though it's not a script :sweat_smile:

I'm not sure exactly what to do for that, either omit it or still add it as part of the skeleton.